client/src/partials/purchases/create/create.html

```diff
@@ -0,0 +1,207 @@
+<div class="flex-header">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.INVENTORY" | translate }}</li>
+      <li class="title">{{ "PURCHASES.PAGE_TITLE" | translate }}</li>
+    </ol>
+  </div>
+</div>
+
+<div class="flex-content">
+  <div class="container-fluid">
+
+    <!--
+      top note panel for the purchase order.details
+      inspired by the patient invoice page's ergonomics
+    -->
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <form name="PurchaseOrderForm" autocomplete="off" autocapitalize="off" autocorrect="off" novalidate>
+              <div class="col-md-6">
+
+                <div class="form-group"
+                  ng-class="{'has-error' : PurchaseOrderForm.$submitted && PurchaseOrderForm.supplier.$invalid }">
+                  <label class="control-label">{{ "FORM.LABELS.SUPPLIER" | translate }}</label>
+                  <input
+                    class="form-control"
+                    ng-model="PurchaseCtrl.supplier"
+                    placeholder="{{ 'FORM.PLACE_HOLDERS.ENTER_SUPPLIER' | translate }}..."
+                    name="supplier"
+                    uib-typeahead="supplier as supplier.name for supplier in PurchaseCtrl.suppliers | filter:$viewValue | limitTo:10"
+                    typeahead-editable="false"
+                    typeahead-on-select="PurchaseCtrl.order.setSupplier(PurchaseCtrl.supplier)"
+                    required>
+                  <div class="help-block" ng-messages="PurchaseOrderForm.supplier.$error" ng-show="PurchaseOrderForm.$submitted">
+                    <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+                  </div>
+                </div>
+
+                <div class="form-group">
```

Comment:
There is no `'has-error'` class here - will these show up highlighted if this is an invalid option in the form?